@xen-orchestra/backups/writers/_MixinBackupWriter.js

```diff
@@ -14,6 +14,10 @@ exports.MixinBackupWriter = (BaseClass = Object) =>
       this._lock = undefined
     }
 
+    cleanVm(options) {
```

Comment:
This is not part of the public writer API, let's use `_` prefix to indicate *protected* visibility.

(Since we can use `#` for private visibility now.)